### PR TITLE
Backend

### PR DESCRIPTION
**Pull request by stevebalk**
Pong game refactoring and some changes to the Makefile.
"make" now get your ips (But at the moment doesn't use them inside Django) and calls docker-compose up -d
also added "make restart" to restart the whole docker-compose setup.

### CODE REVIEW

File: backend/backend/settings.py
Comment (StrongCoding) [inline backend/backend/settings.py:1]: i dont think this is needed for dev
lets use "*" for all hosts and in production hard coded for security

File: backend/usermanagement/migrations/0001_initial.py
Comment (StrongCoding) [inline backend/usermanagement/migrations/0001_initial.py:1]: remove this!

File: .gitignore
Comment (StrongCoding) [inline .gitignore:1]: lets add migrations folder

**Review by StrongCoding — CHANGES_REQUESTED** [review]

File: .gitignore
Comment (stevebalk) [reply .gitignore:1]: Added with 1eb851a

File: backend/backend/settings.py
Comment (stevebalk) [reply backend/backend/settings.py:1]: Changed with 922b61b5d6cb9c8e7ae08f8dce0c8073214510bc

**Review by StrongCoding — APPROVED** [review]

**Pull request merged**